group-3/homeworks/Cvetko_Stavreski/Homework01/script.js

```diff
@@ -0,0 +1,74 @@
+document.getElementById('button').addEventListener("click", function(event) {
+
+// Promise object assigns fulfillment and rejection handlers for asnc manupulated getJson function, or catching the fail reason
+    getJson('filelist.json').then(
+        function(value) {
+
+            let array = JSON.parse(value);
+            let randomNumber = Math.floor(Math.random() * array.length);
+
+            //Use array Nmbering feature to get all names from the innitial array Json file
+            getJson(array[randomNumber]).then(function(object) {
+
+                let result = JSON.parse(object);
+                let arrayNumber = result.data;
+                let operation = result.operation;
+
+                let arraySum = 0;
+                for(let i = 0; i < arrayNumber.length; i++){
+                    arraySum += Operators[operation](arrayNumber[i]);
```

Comment:
Nice touch. 
Also, try to break this kind of code on more than one line, as it can get very confusing to read. This much is ok, but any more, and it would require а lot of thinking to see what the code does.